internal/server/v2/observation/collection.go

```diff
@@ -32,5 +40,96 @@ func FetchFromCollection(
 	childType string,
 	queryDate string,
 ) (*pbv2.ObservationResponse, error) {
-	return nil, nil
+	btData, err := stat.ReadStatCollection(
+		ctx, store.BtGroup, bigtable.BtObsCollection,
+		ancestor, childType, variables, queryDate,
+	)
+	if err != nil {
+		return nil, err
+	}
+	result := &pbv2.ObservationResponse{
+		ObservationsByVariable: map[string]*pbv2.VariableObservation{},
+		Facets:                 map[string]*pb.StatMetadata{},
```

Comment:
nit: Maybe in a future PR, we want to rename "StatMetadata" to "Facet", and rename "MetaHash" to "FaceId"?

Reply:
very good idea! I will make a follow up PR with that